docs/setup/README.md

```diff
@@ -0,0 +1,96 @@
+## Managed Anthos Service Mesh on GKE with Terraform 
+
+This is a general setup using Terraform for the examples in the `docs` folder. 
+
+The following will set up a GKE cluster with Anthos Service Mesh (ASM) with a _Managed Control Plane_ installed . 
+
+The following services will be required for this: 
+* container.googleapis.com
```

Comment:
Do we need to surround these bullet points with `` to have this formatting `container.googleapis.com`?

Reply:
good catch! Fixing